### PR TITLE
MET-5352 Handle empty data / load interactions

### PR DESCRIPTION
**Pull request by andyjmaclean**

### CODE REVIEW

File: projects/sandbox/src/app/progress-tracker/progress-tracker.component.html
Comment (jeortizquan) [inline projects/sandbox/src/app/progress-tracker/progress-tracker.component.html:1]: working...
afbeelding

File: projects/sandbox/src/app/dataset-content-summary/dataset-content-summary.component.html
Comment (jeortizquan) [inline projects/sandbox/src/app/dataset-content-summary/dataset-content-summary.component.html:1]: showing the message
afbeelding

**Review by jeortizquan — APPROVED** [review]
tested in local now by pressing the enter key can query the track progress, orb about loading progress are seen, filter on empty data shows message.

File: projects/sandbox/src/app/_services/sandbox.service.ts [hunk 1]
```diff
@@ -219,7 +219,11 @@ export class SandboxService {
     }
   }
 
-  getDatasetTierSummary(datasetId: string): Observable<DatasetTierSummary> {
-    return of(generateTierSummary(parseInt(datasetId) as number));
+  getDatasetTierSummary(datasetId: number): Observable<DatasetTierSummary> {
+    if (datasetId % 5 === 0) {
+      return of(generateTierSummary(datasetId)).pipe(delay(2000));
```
[the remark this answers is not in the document]
Comment (andyjmaclean) [reply projects/sandbox/src/app/_services/sandbox.service.ts:224]: It isn't, but it's simulating delays to help with testing

**Pull request merged**